### PR TITLE
Create personal_assistant_document.md

### PR DESCRIPTION
**Pull request by Jianfeng777**

### CODE REVIEW

File: xtuner/personal_assistant_document.md [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+
+</details>
+
+### 2.1 环境安装
+首先我们需要先安装一个 XTuner 的源码到本地来方便后续的使用。
+```bash
+# 进入环境后首先 bash
+bash
+# 如果你是在 InternStudio 平台，则从本地 clone 一个已有 pytorch 2.0.1 的环境：
+# /root/share/install_conda_env_internlm_base.sh xtuner0.1.15
```
Comment (KMnO4-zx) [inline xtuner/personal_assistant_document.md:68]: 这里应该取消注释，下面的这一行应该注释叭？不然的话，也没详细说明怎么在非internstudio环境配置torch呀～

> 如果你是在其他平台：conda create --name xtuner0.1.15 python=3.10 -y

Comment (fanqiNO1) [inline xtuner/personal_assistant_document.md:68]: 考虑使用一下开发机提供的 studio-conda 命令？

File: xtuner/personal_assistant_document.md [hunk 2]
```diff
@@ -0,0 +1,711 @@
...
+| --top-p               | 设置累计概率阈值，仅保留概率累加高于top-p的最小标记集，影响生成文本的连贯性 |
+| --seed                | 设置随机种子，用于生成可重现的文本内容                            |
+
+
+除了这些参数以外其实还有一个非常重要的参数就是 `--adapter` ，这个参数主要的作用就是可以在转化后的 adapter 层与原模型整合之前来对该层进行测试。使用这个额外的参数对话的模型和整合后的模型几乎没有什么太多的区别，因此我们可以通过测试不同的权重文件生成的 adapter 来找到最优的 adapter 进行最终的模型整合工作。
+```bash
+# 使用 --adapter 参数与完整的模型进行对话
+xtuner chat /root/ft/model --adapter /root/ft/huggingface
+```
+#### 2.4.3 小结
```
Comment (vansin) [inline xtuner/personal_assistant_document.md:707]: 序号有点问题

File: xtuner/personal_assistant_document.md [hunk 3]
```diff
@@ -0,0 +1,711 @@
...
+在这一节里我们对微调后的模型（adapter）进行了转换及整合的操作，并通过 `xtuner chat` 来对模型进行了实际的对话测试。从结果可以清楚的看出模型的回复在微调的前后出现了明显的变化。那当我们在测试完模型认为其满足我们的需求后，我们就可以对模型进行量化部署等操作了，这部分的内容在之后关于 LMDeploy 的课程中将会详细的进行赘述，这里我们就不多说了。
+
+## 2.5 总结
```
Comment (vansin) [inline xtuner/personal_assistant_document.md:710]: 序号有点问题

File: xtuner/personal_assistant_document.md [hunk 4]
```diff
@@ -0,0 +1,711 @@
...
+3. **PART 3 Dataset & Dataloader**：描述了数据处理的细节，包括如何加载数据集、预处理步骤、批处理大小等，确保了模型能够接收到正确格式和质量的数据。
+
+4. **PART 4 Scheduler & Optimizer**：配置了优化过程中的关键参数，如学习率调度策略和优化器的选择，这些是影响模型训练效果和速度的重要因素。
+
+5. **PART 5 Runtime**：定义了训练过程中的额外设置，如日志记录、模型保存策略和自定义钩子等，以支持训练流程的监控、调试和结果的保存。
+
+一般来说我们需要更改的部分其实只包括前三部分，而且修改的主要原因是我们修改了配置文件中规定的模型、数据集。后两部分都是 XTuner 官方帮我们优化好的东西，一般而言只有在魔改的情况下才需要进行修改。下面我们将根据项目的要求一步步的进行修改和调整吧！
+
+#### 2.3.2 相关路径及参数修改
+首先在 PART 1 的部分，由于我们不再需要在 Huggingface 上自动下载模型，因此我们先要更换模型的路径以及数据集的路径为我们本地的路径。具体操作如下所示：
```
Comment (BaiYu96) [inline xtuner/personal_assistant_document.md:314]: 建议把修改行对应的行号也标注一下，方便学员寻找修改。直接贴diff对于一些没git基础的人来说，可能也还不太好理解。最近有些同学私下问的问题都是因为py文件修改的路径没改对，导致程序无法运行

File: xtuner/personal_assistant_document.md (hunk 1 above)
Comment (zhigexie2015) [reply xtuner/personal_assistant_document.md]: 这里应该把conda安装对应环境的基本命令给罗列下 比如 conda install pytorch==2.0.1之类的 或者至少说明下要通过conda安装哪些主要的包
